docs/docs-content/clusters/edge/site-deployment/deploy-private-registry.md

```diff
@@ -0,0 +1,96 @@
+---
+sidebar_label: "Deploy Cluster with a Private Registry"
+title: "Deploy Cluster with a Private Registry"
+description: "Instructions for creating an Edge Native Cluster Profile"
+hide_table_of_contents: false
+sidebar_position: 60
+tags: ["edge"]
+---
+
+Palette Edge supports authentication with private image registries, which allows your cluster to pull images from a private registry during deployment. You can configure your cluster to pull images from a private registry for both cluster creation and cluster repave. To configure a cluster to pull images from a private image registry, provide the registry URL and the credentials needed to authenticate with the registry in the cluster profile. 
```

Comment:
```suggestion
Palette Edge supports authentication with private image registries, which allows your cluster to pull images from a private registry during deployment. You can configure your cluster to pull images from a private registry for both cluster creation and cluster updates. To configure a cluster to pull images from a private image registry, provide the registry URL and the credentials needed to authenticate with the registry in the cluster profile. 
```

Repaves is a scary word in the k8s word. I think we can replace it with updates as not every cluster update results in a repave.